### PR TITLE
Cloud run pubsub integration

### PR DESCRIPTION
**Pull request by oslaru**
This PR has two main changes:

* Fix object annotation plus truncate policy in Bigquery
* Add Cloud Run Pubsub integration

### CODE REVIEW

File: Dataflow/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.7
```
Comment (kaaquist) [inline Dataflow/Dockerfile:1]: In content ai we use the `3.9-slim-buster`. It is not as large in data size. But I will leave to you to decide if we update.  👍

File: Dataflow/pipeline/requirements_local.txt [hunk 2]
```diff
@@ -9,13 +9,14 @@ docopt==0.6.2
...
 future==0.18.2
+flask==2.0.2
 google-api-core==1.29.0
 google-api-python-client==2.30.0
 google-apitools==0.5.31
 google-auth==1.25.0
 google-cloud-core==1.7.2
 google-cloud-storage==1.42.3
-google-cloud-videointelligence==2.5.1
+google-cloud-videointelligence==1.8.0
```
Comment (kaaquist) [inline Dataflow/pipeline/requirements_local.txt:19]: why are we bumping this here down again?

File: scripts/pubsub-cloud-run-intergration.sh [hunk 3]
```diff
@@ -0,0 +1,231 @@
+# https://bipinupd.medium.com/scheduling-dataflow-pipeline-using-cloud-run-pubsub-and-cloud-scheduler-3d10bd6bdb63
+
+PROJECT_ID=cloud-in-a-box
+PROJECT_NUMBER=612789484517
+REGION=europe-west1
+TOPIC="pubsub-oculi-topic"
+SUBSCRIPTION="pubsub-cloud-run-oculi-subscription"
```
Comment (kaaquist) [inline scripts/pubsub-cloud-run-intergration.sh:7]: let's remove this here and use args instead. (I know it is for local use, but then let's not commit it)
`$/pubsub-cloud-run-intergration.sh -a argument1 -b argument2 -c "argument 3"`
Link: https://unix.stackexchange.com/a/505342

File: scripts/payload.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+	"clientId": "nestle",
+	"brand": "nescafe",
+	"market": "th",
+	"runId": "2021-11-30",
+	"gcsDirPath": "gs://nestle-incn-pr-thvathb2c-th-01-videoai/nestle/nescafe/th/2021-11-30/"
+}
```
Comment (kaaquist) [inline scripts/payload.json:7]: Let's not commit this here. Customer info. Maybe it is time to make the repository private.

**Review by kaaquist — CHANGES_REQUESTED** [review]
A few minor tings.

File: Dataflow/pipeline/requirements_local.txt (hunk 2 above)
Comment (oslaru) [reply Dataflow/pipeline/requirements_local.txt:19]: apache beam is not compatible with 2.5.1 yet,  but oculi doesnt use videointelligence inside apache beam (It calls videointelligence in first step and it starts the beam pipeline once it has all the data), the workaround is to use 1.8.0 in requirements_local.txt and later update to 2.5.1, the way I do in Dockerfile

**Review by kaaquist — APPROVED** [review]

**Pull request merged**